instabilities/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Learning from Training Logbooks
 
-The best learning is to read [Publicly available training logbooks](./resources#publicly-available-training-logbooks) because there you can see exactly what happened and how the problem has been overcome.
+The best learning is to read [Publicly available training logbooks](/resources#publicly-available-training-logbooks) because there you can see exactly what happened and how the problem has been overcome.
```

Comment:
```suggestion
The best learning is to read [Publicly available training logbooks](../resources#publicly-available-training-logbooks) because there you can see exactly what happened and how the problem has been overcome.
```